src/libs/ajax/leonardo/models/disk-models.ts

```diff
@@ -0,0 +1,10 @@
+export interface DiskConfig {
+  name: string
+  size: number
+  //TODO: add type when we type disks
```

Comment:
keeping disk models as simple as possible (this model is relevant for Runtime logic, so defined here)

Reply:
Similar to another comment, ticket #?